packages/console/src/components/Dropdown/index.module.scss

```diff
@@ -0,0 +1,30 @@
+@use '@/scss/underscore' as _;
+
+.content {
+  background: var(--color-on-primary);
+  box-shadow: var(--shadow-light-s2);
+  border-radius: _.unit(2);
+  position: absolute;
+  min-width: 180px;
```

Comment:
`min-width` seems unnecessary to me

Reply:
It looks wired if the width is to small.

Reply:
But 180px seems too big to me. The select component will at least have a downside arrow, right? So technically the minimal width would be 20px (icon) + 2*12 (padding) = 44px. Not that bad to me

Reply:
We might have select buttons like this in our app, so I think we can remove the `min-width` and let the component decide its own width by default.
<img width="196" alt="image" src="https://user-images.githubusercontent.com/12833674/163942322-f4f2c1aa-fdc6-45ad-b681-ade6c65368d9.png">

In cases that you feel the select should be wider, use a CSS class to set the width would do the job.